integration/testcmd/gotest/uinit/gotest.go

```diff
@@ -37,12 +39,50 @@ func walkTests(testRoot string, fn func(string, string)) error {
 	})
 }
 
+var (
+	coverprofile = flag.String("coverprofile", "", "Filename to write coverage data to")
+)
+
+func AppendFile(srcFile, targetFile string) error {
+	cov, err := os.Open(srcFile)
+	if err != nil {
+		return err
+	}
+	defer cov.Close()
+
+	out, err := os.OpenFile(targetFile, os.O_CREATE|os.O_RDWR|os.O_APPEND, 0644)
+	if err != nil {
+		return err
+	}
+	defer out.Close()
+
+	if _, err := io.Copy(out, cov); err != nil {
+		return fmt.Errorf("error appending %s to %s: %v", srcFile, targetFile, err)
+	}
+	return nil
+}
+
 func runTest() error {
-	cleanup, err := common.MountSharedDir()
+	flag.Parse()
+
+	if err := os.MkdirAll("/testdata", 0755); err != nil {
+		log.Fatalf("Couldn't create testdata: %v", err)
+	}
+
+	var (
+		mp  *mount.MountPoint
+		err error
+	)
+	if os.Getenv("UROOT_USE_9P") == "1" {
```

Comment:
Instead, use the existing `MountSharedDir()` function inside of `common/common.go`.